### PR TITLE
Adopt and publish Openshift runbook

### PR DESCRIPTION
**Pull request by taz-pega-work**
Please review the new Openshift runbook and example pega.yaml file I created to edited from the one I used for our deployment on Openshift cluster.

Also fixed a few stray details that needed to be fixed in the GKE runbook.

### CODE REVIEW

File: docs/Deploying-Pega-on-openshift.md [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+- Your Pega-provided SRS Docker image in the repository to which you pushed the image.
+
+- Your choice to enable either:
+
+  - An ElasticSearch service that is defined at [Elasticsearch Helm Chart](https://github.com/elastic/helm-charts/tree/master/elasticsearch) which the SRS provides to your deployment.
+  - Your Elasticsearch cluster for which you specify the required authentication details.
+
+- And additional, supported environmental variables for Elasticsearch basic authentication.
+
+- Three additional parameters that elasticSearch recommends for OpenShift deployments.
```
Comment (pega-kshev) [inline docs/Deploying-Pega-on-openshift.md:181]: If we are adding this statement, could you please provide the link here to https://github.com/elastic/helm-charts/blob/master/elasticsearch/examples/openshift/values.yaml 
Though we might need to update it with exact openshift specific settings later including setting xpack.security.transport.ssl.enabled: false

Comment (taz-pega-work) [reply docs/Deploying-Pega-on-openshift.md]: I have now added these details to the runbook

**Review by pega-kshev — APPROVED** [review]

File: docs/Deploying-Pega-on-openshift.md [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+| elasticsearch: volumeClaimTemplate: resources: requests: storage: | Specify the Elasticsearch cluster disk volume size. Default is 30Gi, set this value to at least three times the size of your estimated search data size | <ul><li>elasticsearch: volumeClaimTemplate: resources: requests: storage:  "\<30Gi>” </li></ul> |
+| Additional OpenShift-required parameters:<ul><li>securityContext:</li><li>podSecurityContext:</li><li>sysctlInitContainer:</li></ul> | Manually add ElasticSearch-recommended parameters to ensure that your SRS pods do not have a RunAsUser parameter and do not require an initialization container. For details, see [OpenShift](https://github.com/elastic/helm-charts/tree/master/elasticsearch/examples/openshift). | <ul><li>securityContext.runAsUser: null</li><li>podSecurityContext.fsGroup: null</li><li>    podSecurityContext.runAsUser: null </li><li>sysctlInitContainer.enabled: false</li></ul> |
```
[the remark this answers is not in the document]
Comment (taz-pega-work) [reply docs/Deploying-Pega-on-openshift.md]: I did this already...you are viewing an outdated push...the current version says this both *both* of the settings and it depends on the users config, right? what if they DO have SSL certs and the will enable it?? I think we have to be generic, right?>>
|elasticsearch: <ul><li>esConfig: elasticsearch.yml: xpack.security.enabled: "true”</li><li>esConfig: elasticsearch.yml: xpack.security.transport.ssl.enabled: "true”</li><li>volumeClaimTemplate: resources: requests: storage:  "\<30Gi>” </li></ul> | To run an internal ElasticSearch service (within your deployment), specify details for both the ElasticSearch security configuration and a cluster disk volume size. While the security pack should be enabled, the transport configuration should match whether an SSL certificate is available; by default it is enabled. For the disk volume size, the default is 30Gi; set this value to at least three times the size of your estimated search data size. | <ul><li>elasticsearch: esConfig: elasticsearch.yml: <ul><li>xpack.security.enabled: "true” </li><li>xpack.security.transport.ssl.enabled: "true” </li></ul></li><li>elasticsearch: volumeClaimTemplate: resources: requests: storage:  "\<30Gi>” </li></ul> |

**Review by pega-kshev — APPROVED** [review]

**Review by pega-kshev — APPROVED** [review]

**Review by MadhuriArugula — APPROVED** [review]

**Review by Anand-Kotamsetti — APPROVED** [review]
LGTM

**Review by Saurabh-16 — APPROVED** [review]

**Pull request merged**